app/wizards/schools/ects/teacher_leaving_wizard/edit_step.rb

```diff
@@ -46,7 +46,7 @@ def leaving_after_start_date
 
           errors.add(
             :leaving_on,
-            "Leaving date must be on or after the start date (#{ect_at_school_period.started_on.to_fs(:govuk)})"
+            "Enter a date which is after when the ECT started at your school (#{ect_at_school_period.started_on.to_fs(:govuk)})"
```

Comment:
As per my other review I think "after when" is an awkward construction and "after the ECT's start date" would be better